spec/board_spec.rb

```diff
@@ -0,0 +1,33 @@
+require './lib/board.rb'
+
+RSpec.describe 'testing the board' do
+  it 'should show us an empty board' do
+    expect(Board.new.show_board).to eq("+------------------------+\n| 1 - ❔  2 - ❔  3 - ❔ | \n| 4 - ❔  5 - ❔  6 - ❔ | \n| 7 - ❔  8 - ❔  9 - ❔ | \n+------------------------+")
+  end
+
+  it 'should show us an empty board' do
+    expect(Board.new.show_board).not_to eq("+------------------------+\n| 1 - p  2 - ❔  3 - ❔ | \n| 4 - ❔  5 - ❔  6 - ❔ | \n| 7 - ❔  8 - ❔  9 - ❔ | \n+------------------------+")
+  end
+
+  context 'update board methods' do
+    it 'should return a board with an input in 5th position' do
+      expect(Board.new.update_board(5, 'x')).not_to eq("+------------------------+\n| 1 - ❔  2 - x  3 - ❔ | \n| 4 - ❔  5 - ❌  6 - ❔ | \n| 7 - ❔  8 - ❔  9 - ❔ | \n+------------------------+")
+    end
+
+    it 'should return a board with an input in 5th position' do
+      expect(Board.new.update_board(5, 'x')).to eq("+------------------------+\n| 1 - ❔  2 - ❔  3 - ❔ | \n| 4 - ❔  5 - ❌  6 - ❔ | \n| 7 - ❔  8 - ❔  9 - ❔ | \n+------------------------+")
+    end
```

Comment:
Both the test statements are the same but both the tests are not. Each test statement should reflect what the actual test is. 

**IMPORTANT!** I have only pointed this out here and above. I have noticed this is the case for most of your tests. So I won't repeat myself. 

- [ ] Please go through all of your tests in the two files and update the test statements so that they reflect the actual test. Eg., the positive scenario test statement should not be the same as the negative scenario test statement.